### PR TITLE
WIP: Feature/design updates

### PR DESCRIPTION
**Pull request by eightfivezero**
In this branch I will push all the improvements from the Gifts polish Notion page (https://www.notion.so/paritytechnologies/Gifts-6d838be326674522923ffabed245b19b#b48eee589ff04c07819ed9ad0937ba65)

So far you can see:
- updated loading indicator (https://github.com/hamidra/dotdrop/issues/134)
- updated background image for Kusama Claim NFT page (when you comment the styles in `_bg.scss` line 12-18 in)
- new icons on about page and gift claim success page
- fixed input focus style on gift details page
- fixed button behaviour on save your seed page

I'm still working on the fading transitions for the card content but this PR could be merged

### CODE REVIEW

File: src/config/development.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (hamidra) [inline src/config/development.json:1]: No need to check-in this provider.

**Comment by hamidra** [comment]
I noticed the addressInfo box in polkadot header is not aligned with the dropdown box. can we also make them aligned or set the adddressInfoBox width to match the dropdown.

Screen Shot 2022-02-13 at 8 50 27 PM

File: public/index.html [hunk 2]
```diff
@@ -1,28 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```
Comment (hamidra) [inline public/index.html:3]: I don't see any changes in the file.  only some indentation mixups. Can be removed from the pull request?!

**Review by hamidra — COMMENTED** [review]
Looks good! Thanks. Can you also fix the style for the addressInfo box

File: public/index.html (hunk 2 above)
Comment (eightfivezero) [reply public/index.html]: I also unstaged the changes here. Must have been some accidental indentation when I opened the file some time! 😅

**Comment by eightfivezero** [comment]
I added some styles for the address info box. Should look good now! :)

**Review by hamidra — APPROVED** [review]

**Pull request merged**